.buildkite/x-pack/pipeline.xpack.osquerybeat.yml

```diff
@@ -297,3 +300,22 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/osquerybeat: Packaging Linux"
+
+      - label: ":ubuntu: x-pack/osquerybeat: Packaging Linux arm64"
+        key: "packaging-arm"
+        env:
+          PLATFORMS: "linux/arm64"
+          PACKAGES: "docker"
```

Comment:
Since we removed `linux/arm64` from the previous step, shouldn't we add `tar.gz` to `PACKAGES` here?
I'd recommend comparing the built artifacts list with the previous packaging configuration

Reply:
:+1

or simply specify `PLATFORMS: +all linux/arm64` and not define `PACKAGES` at all, as we are now doing in the DRA job -> https://github.com/elastic/beats/blob/main/.buildkite/packaging.pipeline.yml#L127

this combo will make it build everything that the specific architecture is allowed to build due to `+all`

Reply:
Good spot!!

Reply:
See [`0bfdc82` (#43019)](https://github.com/elastic/beats/pull/43019/commits/0bfdc828979be73f98c44a963085b1244ddc9ea3)

> I'd recommend comparing the built artifacts list with the previous packaging configuration

Unfortunately, aarch64 was not available for `x-pack/osquerybeat`, see https://github.com/elastic/beats/pull/42705#issuecomment-2698687605

Hence, I don't know where to compare with in this case, if you have any ideas please share